### DISCUSSION TITLE
Music Provider Feature Request Poll #2

### DISCUSSION THREAD

**Discussion by OzGav** [music-assistant/support#1169] [Feature Polls]
It has been nine months since the last poll and a lot has changed. MA V2.0 is in Beta. Soundcloud, Plex, Tidal and Radio Browser have been added as Music Providers and Deezer is under development. So it is time to re-do the poll to find out what the Community would like next!

The existing discussions are not that easy to find and hard to let the devs know what is popular in the Community. So have your say here!

If you would like to discuss these options in more detail then here are the threads. These are in order of the previous vote numbers so lets see if they stay that way!
Apple Music https://github.com/orgs/music-assistant/discussions/389
Amazon Music https://github.com/orgs/music-assistant/discussions/626
Emby https://github.com/orgs/music-assistant/discussions/583
Pandora https://github.com/orgs/music-assistant/discussions/393
WebDAV https://github.com/orgs/music-assistant/discussions/526
Jellyfin https://github.com/orgs/music-assistant/discussions/407
Subsonic https://github.com/orgs/music-assistant/discussions/414
Audible https://github.com/orgs/music-assistant/discussions/862

The poll only takes eight options but here is another contender. Comment and raise your votes there.
SiriusXM https://github.com/orgs/music-assistant/discussions/1161

**Comment by dacmcbibs** [comment]
SiriusXM 😁, just a thought

**Reply by harryfine** [reply]
Agree, SiriusXM streaming service, most Sirius subscribers have this

**Reply by OzGav** [reply]
@dacmcbibs @harryfine Siriusxm is now available in beta 32

**Comment by dacmcbibs** [comment]
Audio book shelf , for those that are into self hosted audio books?

**Reply by DutchJaFO** [reply]
ooh ... off-line/self-hosted anything is always good to have.

**Reply by micha91** [reply]
Oh my dear, I did not know about this solution by now. I will give it a try on a rainy weekend and if it delivers what it promisses, I would be open to write a provider for it. I am currently hosting my audiobooks on plex with audnexus as metadata provider and the Prologue App on iOS, but the overall UX could be better

**Reply by DesignsWithDavid** [reply]
This could be done via the Subsonic API (https://github.com/advplyr/audiobookshelf/issues/399).  As could Jellyfin and Emby.

**Reply by OzGav** [reply]
New devs are always welcome. You can get help starting out on discord.

**Comment by Gessink** [comment]
google podcasts

edit: https://www.cnet.com/tech/googles-podcasts-app-is-dead-you-still-have-time-to-migrate-to-youtube-music/

**Reply by DutchJaFO** [reply]
Spotify also has podcasts ... so I'd suspect we'd need generic podcast support first ?

**Reply by ColtonYYZ** [reply]
They are ending now. So this request isn't going to work out. Sorry

**Comment by spudje** [comment]
I would love to see Bandcamp in here.  The way "Band's Campout" does this in LMS/Squeezebox is awesome. It allows not just to play/stream your Bandcamp collection, but also all Bandcamp music that can be listened to from the Bandcamp website for free.

This is basically the only reason I haven't migrated to a different & more modern local audio streaming server solution.

**Reply by rshomey** [reply]
Seconding Bandcamp!

**Reply by AimPizza** [reply]
Yess, very nice musicians on there :)

**Reply by arexam** [reply]
yay! Bandcamp!

**Comment by wbarber69** [comment]
Spotify Connect

**Comment by alexey2626** [comment]
TuneIn connect

**Comment by volkuhl** [comment]
I don't know if that's possible, but I miss the local audio-in as a provider. It would be nice if I could select my local audio-in of the sound card (Hifiberry) as a provider.
So far, a huge thank you to all developers involved - good job!

**Reply by Paul-Vdp** [reply]
Definitely seconding Local Audio

**Reply by Paul-Vdp** [reply]
VLC would be very nice too

**Reply by OzGav** [reply]
Just want to say there are some existing options if you have a HiFiBerry.

https://www.hifiberry.com/hifiberryos/

https://circuitdigest.com/microcontroller-projects/raspberry-pi-internet-radio-and-streaming-station

For the second option you could use any audio input to the PI which could be a RCA to USB converter (such as https://www.behringer.com/product.html?modelCode=P0484 ). The resulting stream is added to MA as a manual radio station

**Reply by Paul-Vdp** [reply]
@OzGav  I'm afraid I meant Local Audio OUT ...

**Reply by OzGav** [reply]
All good, the OP said audio in. Audio out you should be able to do with the squeezelite addon cant you?

**Reply by Paul-Vdp** [reply]
Got it working after a bit of trial&error. Thanks for the hint - was not aware of the hidden potential in squeezelite ;-)

**Reply by Hedda** [reply]
There is also a related request here asking to stream from a phone audio as source -> https://github.com/orgs/music-assistant/discussions/384

**Comment by freekeys** [comment]
BBC Sounds

**Reply by OzGav** [reply]
BBC Sounds is available in 2.7.0 beta 16 and later

**Comment by paulaltomare** [comment]
A simple Line-In source would be great to take audio from a HiFi and patch it into MA to distribute throughout the house. There are devices out there such as the WiiM Pro that could facilitate this.

**Reply by OzGav** [reply]
That is suggested two posts above this one. You should upvote that one.

**Reply by paulaltomare** [reply]
Just did, thanks.

**Comment by tombloemker** [comment]
I'm a little shocked free options are not a priority over pay options. Iheart Radio and Pandora come to mind.

**Reply by OzGav** [reply]
You can manually add any radio station url

**Comment by sepinho** [comment]
Is there a way to include providers like [Libby](https://libbyapp.com/)? In addition to books and magazines, they're offering plenty of audiobooks.
I guess the (/a) challenge is that you "borrow" audiobooks for a limited time, meaning the actual rights to stream are temporary.

it would be _amazing_ though, because they have a great catalogue and it's included in my local library's services.

**Reply by OzGav** [reply]
You would need to do some research to see if it has an API then find someone to code the provider

**Comment by hackyack8675** [comment]
YouTube Music, and Plex would be nice.

Thanks!

**Reply by OzGav** [reply]
Ummm…. They have been available for quite some time (although Plex is currently unmaintained’

**Comment by daniel-simpson** [comment]
Mixcloud? 🙏

**Reply by OzGav** [reply]
There is an existing feature request in the discussions. Go and upvote there.

**Comment by lpl-maa** [comment]
SiriusXM

**Reply by OzGav** [reply]
You are better off upvoting the existing reply above
